Makefile.PL

```diff
@@ -2,67 +2,103 @@ use warnings;
+            license => [ 'http://dev.perl.org/licenses/' ],
+            x_MailingList => 'http://lists.scsys.co.uk/cgi-bin/mailman/listinfo/dbix-class',
+        },
+        no_index => {
+            directory => [qw(maint xt)],
+        },
+        prereqs => {
+            develop => { requires => \%dev_requires },
+            runtime => { suggests => \%runtime_suggests },
+        },
+    },
 );
-my $args = {
-    skip_author_deps => undef,
-};
-$getopt->getoptions($args, 'skip_author_deps');
-if (@ARGV) {
-    warn "\nIgnoring unrecognized option(s): @ARGV\n\n";
-}
-
-use FindBin;
-use lib "$FindBin::Bin/lib";
-
-perl_version '5.008001';
-name           'DBIx-Class-Schema-Loader';
-all_from       'lib/DBIx/Class/Schema/Loader.pm';
-
-# temporary, needs to be thrown out
-test_requires 'DBIx::Class::IntrospectableM2M' => 0;
 
-# core, but specific versions not available on older perls
-test_requires 'File::Temp'      => '0.16';
-test_requires 'File::Path'      => '2.07';
-
-test_requires 'DBD::SQLite'     => '1.29';
-test_requires 'Test::Exception' => '0.31';
-test_requires 'Test::More'      => '0.94';
-test_requires 'Test::Warn'      => '0.21';
-test_requires 'Test::Deep'      => '0.107';
-test_requires 'Test::Differences' => '0.60';
-
-requires 'Carp::Clan'                  => 0;
-requires 'Class::Accessor::Grouped'    => '0.10008';
-requires 'Class::C3::Componentised'    => '1.0008';
-requires 'Class::Inspector'            => '1.27';
-requires 'Class::Unload'               => '0.07';
-requires 'curry'                       => '1.000000',
-requires 'Data::Dump'                  => '1.06';
-requires 'DBIx::Class'                 => '0.08127';
-requires 'Hash::Merge'                 => '0.12';
-requires 'Lingua::EN::Inflect::Number' => '1.1';
-requires 'Lingua::EN::Tagger'          => '0.23';
-requires 'Lingua::EN::Inflect::Phrase' => '0.15';
-requires 'List::Util'                  => '1.33';
-requires 'MRO::Compat'                 => '0.09';
-requires 'namespace::clean'            => '0.23';
-requires 'Scope::Guard'                => '0.20';
-requires 'String::ToIdentifier::EN'    => '0.05';
-requires 'String::CamelCase'           => '0.02';
-requires 'Sub::Util'                   => '1.40';
-requires 'Try::Tiny'                   => 0;
-
-# core, but specific versions not available on older perls
-requires 'Digest::MD5'                 => '2.36';
-requires 'Exporter'                    => '5.63';
+sub _move_to {
+  my ($hash, $fromkey, $tokey) = @_;
+  $hash->{$tokey} = {
+    %{ $hash->{$tokey} || {} },
+    %{ delete($hash->{$fromkey}) || {} },
+  };
+}
+delete $eumm_args{META_MERGE} if $eumm_version < 6.45_01;
+_move_to(\%eumm_args, 'CONFIGURE_REQUIRES', 'PREREQ_PM')
+  if $eumm_version < 6.51_03;
+_move_to(\%eumm_args, 'BUILD_REQUIRES', 'PREREQ_PM')
+  if $eumm_version < 6.55_01;
+$eumm_args{NO_MYMETA} = 1
+  if $eumm_version >= 6.57_02 and $eumm_version < 6.57_07;
+_move_to(\%eumm_args, 'TEST_REQUIRES', 'PREREQ_PM')
```

Comment:
This should move the prereqs to `BUILD_REQUIRES` and be moved before `BUILD_REQUIRES` gets conditionally moved to `PREREQ_PM`.

Reply:
Thanks, force-pushed!